### PR TITLE
Doc - Update data node filter section

### PR DESCRIPTION
**Pull request by trgiangdo**

### CODE REVIEW

File: docs/manuals/core/entities/data-node-mgt.md [hunk 1]
```diff
@@ -1163,29 +1163,201 @@ Correspondingly, In memory data node can write any data object that is valid dat
...
+or a list of operators to the `DataNode.filter()^` method.
 
-```python linenums="1"
-data_node.filter(
-    [("field_name", 14, Operator.EQUAL), ("field_name", 10, Operator.EQUAL)],
-    JoinOperator.OR
-)
+Assume that the content of the data node can be represented by the following table.
+
+!!! example "Example data"
```
Comment (jrobinAV) [inline docs/manuals/core/entities/data-node-mgt.md]:
```suggestion
!!! example "Data sample"
```

File: docs/manuals/core/entities/data-node-mgt.md [hunk 2]
```diff
@@ -1163,29 +1163,201 @@ Correspondingly, In memory data node can write any data object that is valid dat
...
+
+    | date       | nb_sales |
+    |------------|----------|
+    | 12/24/2018 | 1550     |
+    | 12/25/2018 | 2315     |
+    | 12/26/2018 | 1832     |
+
+In the following example, the `DataNode.filter()^` method will return all the records from the data node
+where the value of the "nb_sales" field is equal to 1500.
```
Comment (jrobinAV) [inline docs/manuals/core/entities/data-node-mgt.md]:
```suggestion
where the value of the "nb_sales" field is equal to 1550.
```

File: docs/manuals/core/entities/data-node-mgt.md [hunk 3]
```diff
@@ -1163,29 +1163,201 @@ Correspondingly, In memory data node can write any data object that is valid dat
...
+!!! example "Filter data where "nb_sales" is equal to 1550"
+
+    === "exposed_type = "pandas""
+
+        ```python
+        filtered_data = pandas.DataFrame
+        (
+                     date  nb_sales
+            0  12/24/2018      1550
+        )
```
Comment (jrobinAV) [inline docs/manuals/core/entities/data-node-mgt.md]: It is strange to me to use Python assignment operator to explain the value of a variable.
Should we replace the '=' assignment with an '==' operator?
Or maybe in the previous code, we could print the output of `data_node.filter()` and show the console output.

Comment (trgiangdo) [reply docs/manuals/core/entities/data-node-mgt.md]: Let me propose something to improve this

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**